### DISCUSSION TITLE
SQL PROJECT

### DISCUSSION THREAD

**Discussion by Balqeesyusuf** [SeaQL/summer-of-code#9] [General]
Excited to work on an existing project.

**Comment by tyt2y3** [comment]
Hi! Application now opens so your should draft and submit your proposal now so that we will have the chance of review it early and give comments! https://github.com/SeaQL/summer-of-code/blob/main/2022/CONTRIBUTING.md

**Comment by billy1624** [comment]
Hey @Balqeesyusuf, welcome! Please submit your early so we can review and provide feedbacks :)
